src/zenml/zen_stores/schemas/flavor_schemas.py

```diff
@@ -68,6 +68,8 @@ class FlavorSchema(NamedSchema, table=True):
     )
     user: Optional["UserSchema"] = Relationship(back_populates="flavors")
 
+    logo_url: str
```

Comment:
Can we perhaps make this an optional parameter?

Reply:
sure